.travis.yml

```diff
@@ -61,4 +63,4 @@ script:
     fi
 
 after_success:
-  pipenv run coveralls
+  pipenv run coveralls
```

Comment:
this line should not have been changed, configure your editor to always add a new line at the end of the file, edit this file again (ensure there's a new line at the end) and amend the commit with `git commit --amend`, make sure this line is unchanged.

Please read carefully our contributing guidelines, especially the following section: http://openwisp.io/docs/developer/contributing.html#avoiding-unnecessary-changes

Reply:
Ok

Reply:
Fixed it